### PR TITLE
Fixlinks2: Fixes to internal anchors

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: en/config_mc/mc_slew_rate_type_trajectory.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ This parameter is used for position-control in the horizontal direction, where t
 The limit for the rate of change of the velocity setpoint is defined by [MPC_ACC_HOR_MAX](../advanced_config/parameter_reference.md#MPC_ACC_HOR_MAX).
 This parameter should be set larger than any of the other acceleration related parameters in the horizontal direction.
 
-<span id="mpc_acc_hor-and-mpc_dec_hor_slow"></span>
+<a id="mpc_acc_hor-and-mpc_dec_hor_slow"></a>
 #### MPC_ACC_HOR and MPC_DEC_HOR_SLOW
 
-In velocity-control the rate limit for the velocity setpoint is extracted from a linear map from stick input to acceleration limit with maximum [MPC_ACC_HOR](../advanced_config/parameter_reference.md#MPC_ACC_HOR) and minimum [MPC_DEC_HOR_SLOW](../advanced_config/parameter_reference.md#MPC_DEC_HOR_SLOW).
+In velocity-control the rate limit for the velocity setpoint is extracted from a linear map from stick input to acceleration limit with maximum [MPC_ACC_HOR](../advanced_config/parameter_reference.md#MPC_ACC_HOR) and minimum [MPC_DEC_HOR_SLOW](../advanced_config/parameter_reference.md#MPC_DEC_HOR_SLOW).
```
Comment (hamishwillee) [inline en/config_mc/mc_slew_rate_type_trajectory.md:40]: @MaEtUgR `MPC_DEC_HOR_SLOW` appears to have been removed. How should this section (and other mentions in this doc change?)

File: en/config_mc/mc_slew_rate_type_trajectory.md [hunk 2]
```diff
@@ -78,5 +78,5 @@ These two parameters only have effect during the transition from **velocity-cont
 The purpose of these two parameters are to minimize the jerk introduced from forward flight to hover (please see [MPC_ACC_HOR and MPC_DEC_HOR_SLOW](#mpc_acc_hor-and-mpc_dec_hor_slow)).
 
 The jerk-parameter controls the rate limit with which the acceleration limit can change to `MPC_ACC_HOR_MAX`.
-The actual jerk-value is a linear map from velocity speed to jerk where full speed maps to [MPC_JERK_MAX](../advanced_config/parameter_reference.md#MPC_JERK_MAX) and zero speed to [MPC_JERK_MIN](../advanced_config/parameter_reference.md#MPC_JERK_MIN).
+The actual jerk-value is a linear map from velocity speed to jerk where full speed maps to [MPC_JERK_MAX](../advanced_config/parameter_reference.md#MPC_JERK_MAX) and zero speed to [MPC_JERK_MIN](../advanced_config/parameter_reference.md#MPC_JERK_MIN).
```
Comment (hamishwillee) [inline en/config_mc/mc_slew_rate_type_trajectory.md:81]: @MaEtUgR `MPC_JERK_MIN` appears to have been removed. How should this section (and other mentions in this doc change?)

**Comment by hamishwillee** [comment]
Merging. There are open questions around jerkmin and friends above: https://github.com/PX4/PX4-user_guide/pull/1305#discussion_r644570915

File: en/config_mc/mc_slew_rate_type_trajectory.md (hunk 1 above)
Comment (MaEtUgR) [reply en/config_mc/mc_slew_rate_type_trajectory.md:40]: The position mode implementation (flight task) that used this was removed in https://github.com/PX4/PX4-Autopilot/pull/16052
So the entire slew-rate-based trajectory is deprecated.

File: en/config_mc/mc_slew_rate_type_trajectory.md (hunk 2 above)
Comment (MaEtUgR) [reply en/config_mc/mc_slew_rate_type_trajectory.md:81]: Same as https://github.com/PX4/PX4-user_guide/pull/1305#discussion_r644662752

**Pull request merged**